src/ui/pages/SendForm/SuccessState/SuccessState.tsx

```diff
@@ -16,8 +16,37 @@ import { Button } from 'src/ui/ui-kit/Button';
 import { HStack } from 'src/ui/ui-kit/HStack';
 import { TextAnchor } from 'src/ui/ui-kit/TextAnchor';
 import { NavigationTitle } from 'src/ui/components/NavigationTitle';
+import { FEATURE_LOYALTY_FLOW } from 'src/env/config';
 import { TransferVisualization } from '../TransferVisualization';
 
+export function GasbackDecorated({ value }: { value: number }) {
+  return (
+    <HStack
+      gap={8}
+      justifyContent="space-between"
+      alignItems="center"
+      style={{
+        padding: '8px 12px',
+        borderRadius: 12,
+        background:
+          'linear-gradient(90deg, rgba(160, 36, 239, 0.20) 0%, rgba(253, 187, 108, 0.20) 100%)',
+      }}
+    >
+      <UIText kind="small/regular">Gasback</UIText>
+      <UIText
+        kind="small/accent"
+        style={{
+          background: 'linear-gradient(90deg, #6C6CF9 0%, #FF7583 100%)',
```

Comment:
May be we need move this value to a global var?

Reply:
it's not like we have a design system that consistently uses these gradients...